connectors/src/connectors/notion/index.ts

```diff
@@ -273,6 +296,12 @@ export class NotionConnectorManager extends BaseConnectorManager<null> {
       return new Err(new Error("Connector not found"));
     }
 
+    // delete the "Orphaned Resources" folder
+    await deleteDataSourceFolder({
```

Comment:
we can discuss this, i think to be consistent with the rest of the codebase, core stuff should not be deleted here; I'd say this can be tackled as part of the upcoming https://github.com/dust-tt/dust/issues/9511

Reply:
Thinking about it again I think we should do it in front actually, but we should def leave it for the task

Reply:
removed the delete from here, added a cmment in the task to not forget about it